### DISCUSSION TITLE
Live demos don't work

### DISCUSSION THREAD

**Discussion by akinuri** [LavaLite/cms#412] [General]
The [live demos](https://demo.lavalite.org/admin/login) don't work. When I try to log in, I get an error. Are the demo accounts deleted or changed?

> These credentials do not match our records.

**Comment by cuskar** [comment]
> The [live demos](https://demo.lavalite.org/admin/login) don't work. When I try to log in, I get an error. Are the demo accounts deleted or changed?
> 
> > These credentials do not match our records.

Someone delete it and change into ads site
